compair/models/course.py

```diff
@@ -18,6 +18,7 @@ class Course(DefaultTableMixin, UUIDMixin, ActiveMixin, WriteTrackingMixin):
     name = db.Column(db.String(255), nullable=False)
     year = db.Column(db.Integer, nullable=False)
     term = db.Column(db.String(255), nullable=False)
+    sandbox = db.Column(db.SmallInteger(), nullable=False, server_default='0', default=0)
```

Comment:
you can use`db.Boolean(name='sandbox')` (I think it works a bit better across different databases and also uses `TINYINT` for mysql). Would need to upgrade the migration as well